testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/pages/AuthenticationMethodSetupPage.java

```diff
@@ -0,0 +1,54 @@
+package org.keycloak.testsuite.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+/**
+ * Setup page for various authentication methods(Password/OTP/RecoveryAuthnCodes etc.)
+ *
+ * @author <a href="mailto:vnukala@redhat.com">Venkata Nukala</a>
+ */
+public class AuthenticationMethodSetupPage extends LanguageComboboxAwarePage {
```

Comment:
This class is used in RecoveryAuthnCodesAuthenticatorTest, but this test does not use it for anything. So since this class is effectively not used anywhere, it can be possible removed from this PR?